.azure-pipelines/livetest.yml

```diff
@@ -0,0 +1,9 @@
+jobs:
+  - job:
+    pool:
+      vmImage: 'Ubuntu 18.04'
+    steps:
+      - template: steps/init_workspace.yml
+      - template: steps/linter_check.yml
+      - template: steps/build_test.yml
```

Comment:
You can remove these two steps, since this stage runs when the first stage succeeds, the linter and build should passed.

Reply:
removed